### PR TITLE
Define color palette & tokens for theming

### PR DESCRIPTION
**Pull request by SerhiiTsybulskyi**
- Defined color palette
- Defined color tokens
- Updated class names for changed tokens

I would suggest merge light/dark theme in one theme and manage color via TW config with [tw-color](https://www.npmjs.com/package/tw-colors) plugin.

Alternative: [Tailwindcss Themer](https://www.npmjs.com/package/tailwindcss-themer)

### CODE REVIEW

File: src/elements/Chip/ChipTheme.ts [hunk 1]
```diff
@@ -117,53 +117,53 @@ export const lightChipTheme: ChipTheme = {
...
-      base: 'bg-primary-700 border-primary-700 text-primary-700',
+      base: 'bg-primary border-primary text-primary',
       selectable: [
         baseTheme.colors.primary.selectable,
-        'hover:bg-primary-600 hover:border-primary-600'
+        'hover:bg-primary-hover hover:border-primary-hover'
       ].join(' '),
-      selected: '!bg-primary-600 text-white',
-      focus: 'focus-visible:outline-primary-500'
+      selected: '!bg-primary text-white',
```
Comment (SerhiiTsybulskyi) [inline src/elements/Chip/ChipTheme.ts:125]: We have selected state for Chip, but idk what token we should use for this.
currently "active" token has same color with default.

File: tailwind.config.ts [hunk 2]
```diff
@@ -55,25 +194,71 @@ const config: Config = {
...
+        },
+        background: {
+          level1: colorPalette.white,
+          level2: colorPalette.gray['01'],
+          level3: colorPalette.gray['02'],
+          level4: colorPalette.gray['03'],
+        },
+        panel: newColors['black-pearl'],
+        surface: newColors['charade'],
+        typography: {
```
Comment (amcdnl) [inline tailwind.config.ts:243]: Is this a standard field?

File: tailwind.config.ts [hunk 3]
```diff
@@ -55,25 +194,71 @@ const config: Config = {
...
+          DEFAULT: newColors['athens-gray'],
+        },
+        accent: {
+          DEFAULT: newColors['waterloo'],
+          active: newColors['anakiwa']
+        },
         disabled: colors.gray['400'], // text-disabled
         light: {
```
Comment (amcdnl) [inline tailwind.config.ts:251]: should these be moved to tw-colors now?

File: tailwind.config.ts [hunk 4]
```diff
@@ -2,38 +2,177 @@ import { type Config } from 'tailwindcss';
...
-// https://tailwindcss.com/docs/theme#referencing-other-values
-const newColors = {
-  'blue-ribbon': {
-    // primary
-    DEFAULT: '#105EFF'
+const colorPalette = {
+  white: '#FFFFFF',
+  black: '#000000',
+  gray: {
+    '01': '#F2F3F7',
```
Comment (amcdnl) [inline tailwind.config.ts:9]: need to update these color names

File: tailwind.config.ts (hunk 2 above)
Comment (SerhiiTsybulskyi) [reply tailwind.config.ts:243]: It's my vision to name colors for text.
Figma has `text-01, text-02, text-03` currently
I've shared my vision with Lisa to rethink how to mane it and make it like
```
typography: {
  title: 'color',
  label: 'color'
  body: 'color'
  button: 'color'
  etc...
}
```

**Pull request merged**